packages/bpk-component-switch/README.md

```diff
@@ -0,0 +1,36 @@
+# bpk-component-switch
+
+> Backpack switch component.
+
+## Installation
+
+```sh
+npm install bpk-component-switch --save-dev
+```
+
+## Usage
+
+```js
+import React from 'react';
+import BpkSwitch from 'bpk-component-code';
+
+export default () => (
+  <div>
+    <BpkSwitch label="Switched on" checked />
+    <BpkSwitch label="Turned off" />
+  </div>
+)
+```
+
+## Props
+
+| Property  | PropType | Required | Default Value |
+| --------- | -------- | -------- | ------------- |
+| label     | Node     | true     | -             |
+| className | string   | false    | null          |
+
+This component uses a hidden checkbox, so it supports all the same properties as it (for example `checked`).
```

Comment:
Might be worth linking through to the Checkbox props here for reference/when on the docs site?

Reply:
It's not a `BpkCheckbox`, just a regular checkbox. But I'll make that more explicit and link to MDN.